spec/presenters/email_alert_presenter_spec.rb

```diff
@@ -31,8 +33,16 @@
         expect(presented_data[:tags][:format]).to eq(cma_case_payload["document_type"])
         expect(presented_data[:tags][:case_type]).to eq(cma_case_payload["details"]["metadata"]["case_type"])
         expect(presented_data[:document_type]).to eq(cma_case_payload["document_type"])
+        expect(presented_data[:public_updated_at]).to include(cma_case_payload["public_updated_at"])
+        expect(presented_data[:base_path]).to include(cma_case_payload["base_path"])
         expect(presented_data[:footer]).to include("SUBSCRIBER_PREFERENCES_URL")
         expect(presented_data[:header]).to include("govuk-email-header")
+
+        expect(presented_data.keys.map(&:to_s)).to match_array(%w(
+          title description change_note subject body tags document_type
+          email_document_supertype government_document_supertype content_id
+          public_updated_at publishing_app base_path urgent header footer
+        ))
```

Comment:
might be a little cleaner to drop the map and use `%i` instead of `%w`

Reply:
Good point, fixed!